code/modules/mob/living/simple_animal/hostile/spider_queen.dm

```diff
@@ -116,3 +124,7 @@
 				M.apply_damage(50, DAMAGE_BRUTE)
 				M.apply_effect(6, STUN, blocked)
 	return TRUE
+
+/mob/living/simple_animal/hostile/giant_spider/nurse/spider_queen/Life()
+	..()
+	adjustBruteLoss(-5)
```

Comment:
undocumented change, and seems excessive to me

Reply:
The queen has no other way to regenerate health. Atm I'm using the same regen rate the cavern geist does. Would you prefer -3?

Reply:
Changed it to -3